infrastructure-playbooks/storage-inventory.yml

```diff
@@ -0,0 +1,36 @@
+---
+# This playbook queries each OSD using `ceph-volume inventory` to report the
+# entire storage device inventory of a cluster.
+#
+# Usage:
+#   ansible-playbook storage-inventory.yml
+
+- name: gather facts and check the init system
+
+  hosts:
+    - "{{ osd_group_name|default('osds') }}"
+
+  become: true
+
+  tasks:
+    - debug: msg="gather facts on all Ceph hosts for following reference"
+
+- name: query each host for storage device inventory
+
+  hosts:
+    - "{{ osd_group_name|default('osds') }}"
+
+  become: true
+
+  tasks:
+    - import_role:
+        name: ceph-defaults
+
+  post_tasks:
```

Comment:
the `post_tasks` is not really needed here, you can simply put this task as part of `tasks` above.